profile-models/src/main/java/com/telecominfraproject/wlan/profile/rf/models/RfElementConfiguration.java

```diff
@@ -0,0 +1,413 @@
+package com.telecominfraproject.wlan.profile.rf.models;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.telecominfraproject.wlan.core.model.equipment.AutoOrManualValue;
+import com.telecominfraproject.wlan.core.model.equipment.ChannelBandwidth;
+import com.telecominfraproject.wlan.core.model.equipment.ChannelHopSettings;
+import com.telecominfraproject.wlan.core.model.equipment.RadioBestApSettings;
+import com.telecominfraproject.wlan.core.model.equipment.RadioType;
+import com.telecominfraproject.wlan.core.model.json.BaseJsonModel;
+import com.telecominfraproject.wlan.equipment.models.ActiveScanSettings;
+import com.telecominfraproject.wlan.equipment.models.ManagementRate;
+import com.telecominfraproject.wlan.equipment.models.MimoMode;
+import com.telecominfraproject.wlan.equipment.models.MulticastRate;
+import com.telecominfraproject.wlan.equipment.models.NeighbouringAPListConfiguration;
+import com.telecominfraproject.wlan.equipment.models.StateSetting;
+
+public class RfElementConfiguration extends BaseJsonModel {
+
+	private static final long serialVersionUID = 1071268246965238451L;
+	
+    private static final Logger LOG = LoggerFactory.getLogger(RfElementConfiguration.class);
+    
+    public final static int MIN_BG_RADIO_CELL_SIZE = -80;
```

Comment:
Can MIN_XX_RADIO_CELL_SIZE be made private? Otherwise, if we're going to have rf-profile per radio type then I think these public values should be per radio type, not ac/bg

Reply:
Sure, I'll update those fields